chapters/05-version-control-solo.Rmd

```diff
@@ -12,22 +12,49 @@ Clearly identify which tasks are best accomplished by each interface.
   - adding and updating remotes
 - Setting up GitHub for R package (make use of `usethis`), pushing and pulling
 
-## Final exercise
+<!---
+Moved from syllabus:
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+And some people still worry that if they make their data and code generally available,
+someone else will use it and publish a result they have come up with themselves.
+This is almost unheard of in practice,
+but that doesn't stop people using it as a scare tactic.
+Other people are afraid of looking foolish or incompetent by sharing code that might contain bugs.
+This isn't just [impostor syndrome][impostor-syndrome]:
+members of marginalized groups are frequently judged more harshly than others,
+so being wrong in public is much riskier for them.
+-->
 
-1. For the project previously used for demonstration in this chapter,
+## Final exercise {#05-final-exercise}
+
+Complete these tasks as part of the package for your first project assignment.
+Refer to the project assignment for exact details on what you are required to
+create and on what the package should do. As with the other final exercises,
+we will use the `kenyaweather` package as an example and reference.
+
+TODO: This question makes a change to zipfs, so this should be reflected in any automation we do to it.
+
+1. For the `zipfs` previously used for demonstration in this chapter,
```

Comment:
Given how the other chapters are written, the activities in this first question will be covered as demos in the actual chapter, correct?

Reply:
Yes, that's right.